docs/source/conf.py

```diff
@@ -45,8 +45,7 @@
     "sphinx.ext.intersphinx",
     "sphinx.ext.viewcode",
     "sphinx.ext.napoleon",
-    "sphinx_js",
-    "recommonmark",
```

Comment:
Yes, it had been removed from setup.py because of severe dependency issues. I now created following issue to possibly re-add the package: https://github.com/Dallinger/Dallinger/issues/4992 . See also this PR: https://github.com/Dallinger/Dallinger/pull/4644